coalib/testing/BaseTestHelper.py

```diff
@@ -0,0 +1,41 @@
+from coala_utils.Comparable import Comparable
+
+
+class BaseTestHelper:
+    """
+    This is a base class for all Bears' tests of coala's testing API.
+    """
+
+    def assertComparableObjectsEqual(self, observed_result, expected_result):
```

Comment:
Perhaps you could use snake case for the method name, to ensure consistency. `assert_comparable_objects_equal`

Reply:
The method name sounds a bit complicated. Is there some way it can be made a bit more intuitive perhaps?

Reply:
I just gone through the yash Nishar commit of `assertComparableObjectsEqual` and I didn't find any review by John or Makman on name casing, btw for me `assertComparableObjectsEqual` is ok here.

Sorry for nitpicking the comment :stuck_out_tongue:

Reply:
@RaiVaibhav, it is not necessary to keep same thing in the base class. We can think of better method name here as you can see the previous method name is similar to unittest method name type. But in this project, we are moving towards pytest.